src/commands/lambda/functions/commons.ts

```diff
@@ -501,3 +502,37 @@ export const willUpdateFunctionConfigs = (configs: FunctionConfiguration[]) => {
 
   return willUpdate
 }
+
+export const maskEnvVar = (key: string, value: string) => {
+  if (SKIP_MASKING_ENV_VARS.has(key)) {
+    return value
+  }
+
+  // Don't mask booleans
+  if (value.toLowerCase() === 'true' || value.toLowerCase() === 'false') {
+    return value
+  }
+
+  // Dont mask numbers
+  if (!isNaN(Number(value))) {
+    return value
+  }
+
+  // Mask entire string if it's short
+  if (value.length < 12) {
+    return '*'.repeat(16)
+  }
+
+  // Keep first two and last four characters if it's long
+  return value.slice(0, 2) + '*'.repeat(10) + value.slice(-4)
+}
+
+export const maskUpdateFunctionConfigurationCommandInput = (config: UpdateFunctionConfigurationCommandInput) => {
```

Comment:
I'm having trouble understanding this function. Could you document how it works?

Reply:
I did a minor refactor and added some comments documenting the intended use of this function. Let me know if I should clarify further!